### ISSUE TITLE
Translate to Hindi

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns#2279]
This task tracks the translation of the patterns. More information about the translation process is available at https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

Acceptance criteria

- The patterns have been translated
- The patterns can be browsed in Github and on the website https://java-design-patterns.com

**Comment by AnantGupta001** [comment]
Hey, can you assign this issue to me?

**Comment by Harsh4902** [comment]
Hey, I am from India and I want to help your for translating this repo in Hindi. How can I do that??

**Comment by iluwatar** [comment]
@Harsh4902 maybe this will get you started: https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

See the existing translations under `localization` for concrete examples. Thanks for your help in advance!

**Comment by Ayush-Thakur-geek** [comment]
Hey, I am from India and I would like to contribute to this (as my first contribution) if no one else is working on it.

**Comment by iluwatar** [comment]
Yes, sorry for the late reply @Ayush-Thakur-geek Please go ahead!

**Comment by Ayush-Thakur-geek** [comment]
Hey @iluwatar did 2 prs.
Could you pls. review and approve.

**Comment by surjendu104** [comment]
Hi @iluwatar I want to add hindi translation for some of the design patterns. Could you assign it to me.

**Comment by surjendu104** [comment]
@iluwatar I have made one PR. ps check it out
